lib/components/artist/related_artists/related_artist.js

```diff
@@ -3,25 +3,32 @@
 
 import Relay from 'react-relay';
 import React from 'react-native';
-const { ScrollView, StyleSheet, View, Text, Dimensions } = React;
+const { ScrollView, StyleSheet, View, Text, TouchableWithoutFeedback, Dimensions } = React;
 
 import colors from '../../../../data/colors';
 import SerifText from '../../text/serif';
 import ImageView from '../../opaque_image_view';
+import SwitchBoard from '../../../modules/switch_board';
 
 const windowDimensions = Dimensions.get('window');
 const imageWidth = windowDimensions.width > 700 ? (windowDimensions.width - 100)/3 : (windowDimensions.width - 60)/2;
 
 class RelatedArtist extends React.Component {
+  handleTap() {
+    SwitchBoard.presentNavigationViewController(this, `/artist/${this.props.artist._id}`);
+  }
```

Comment:
Seems like metaphysics doesn’t provide slugs for most models, the real DB ID should work too, though.